src/pty.c

```diff
@@ -134,6 +134,12 @@
  */
 #ifndef O_NOCTTY
 # define O_NOCTTY 0
+#endif
+
+#if defined(HAVE_SVR4_PTYS) || defined(HAVE_POSIX_OPENPT)
+char *ptsname(int);
+int unlockpt(int);
+int grantpt(int);
```

Comment:
Why are we declaring those functions instead of doing `#include <stdlib.h>`?
Doing such declarations could be dangerous if they don't match what's on `stdlib.h`.

Reply:
Linux needs `_XOPEN_SOURCE` to declare them in `stdlib.h`.